src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -175,9 +177,15 @@ public void MultitaskingControl_CurrentInstanceChanged(object? sender, CurrentIn
 				SidebarAdaptiveViewModel.PaneHolder.PropertyChanged -= PaneHolder_PropertyChanged;
 
 			var navArgs = e.CurrentInstance.TabItemParameter?.NavigationParameter;
-			SidebarAdaptiveViewModel.PaneHolder = e.CurrentInstance as IPaneHolder;
+			var currentInstance = e.CurrentInstance as IPaneHolder;
+			if (currentInstance is null || navArgs is null) 
+   				return;
```

Comment:
You can use pattern matching.

```suggestion
			if (e.CurrentInstance is not IPaneHolder currentInstance || navArgs is null) 
   				return;
```